react/src/navigation/navigator.tsx

```diff
@@ -187,15 +187,15 @@ export function urlFromPageDescriptor(pageDescriptor: ExceptionalPageDescriptor)
         case 'article':
             pathname = '/article.html'
             searchParams = {
-                longname: sanitize(pageDescriptor.longname),
```

Comment:
@kavigupta This is going to break some recently shared links (it was a mistake in the implementation). Only those that need to be sanitized though, which should be few

Reply:
Yeah that's fine